src/microbe_stage/MicrobeInternalCalculations.cs

```diff
@@ -253,56 +253,22 @@ public static float CalculateCapacity(IEnumerable<OrganelleTemplate> organelles)
     /// <returns>The rotation speed</returns>
     public static float CalculateRotationSpeed(PhysicsShape shape, IEnumerable<IPositionedOrganelle> organelles)
     {
-        // Calculate rotation speed based on the inertia.
-        var speedFraction = shape.TestYRotationInertiaFactor();
-
-        // TODO: a good non-linear function here
-        // TODO: also should update MicrobeColonyHelpers.CalculateRotationMultiplier as that uses the same approach as
-        // here right now
-        // GD.Print("Speed fraction: ", speedFraction);
-
-        // if (speedFraction < 0.5284f)
-        // {
-        //     speedFraction = Mathf.Pow(1.1f * speedFraction - 0.8f, 3) + 0.5f;
-        // }
-        // else
-        // {
-        //     speedFraction = Mathf.Pow(1.3f * speedFraction - 0.5f, 3) + 0.483f;
-        // }
-
-        // This seems pretty much, fine though the top end is pretty harsh so at least for the top end another function
-        // should be used to get better small cell scaling)
-        speedFraction = Mathf.Clamp(Mathf.Pow(speedFraction, 1.0f / 4.0f), 0, 1);
-
-        var speed = Constants.CELL_MAX_ROTATION -
-            ((Constants.CELL_MAX_ROTATION - Constants.CELL_MIN_ROTATION) * speedFraction);
-
-        int ciliaCount = 0;
-
-        // For simplicity we calculate all cilia af if they are at a uniform (max radius) distance from the center
-        float radiusSquared = 1;
+        float cilliaFactor = 0;
 
         foreach (var organelle in organelles)
         {
-            if (!organelle.Definition.HasCiliaComponent)
-                continue;
-
-            ++ciliaCount;
-
-            var distance = Hex.AxialToCartesian(organelle.Position).LengthSquared();
-
-            if (radiusSquared < distance)
-                radiusSquared = distance;
+            if (organelle.Definition.HasCiliaComponent)
+            {
+                cilliaFactor += Constants.CILIA_ROTATION_FACTOR +
+                    Hex.AxialToCartesian(organelle.Position).LengthSquared() *
+                    Constants.CILIA_RADIUS_FACTOR_MULTIPLIER;
+            }
         }
 
-        // Add the extra speed from cilia
-        if (ciliaCount > 0)
-        {
-            speed -= ciliaCount * Mathf.Sqrt(radiusSquared) * Constants.CILIA_RADIUS_FACTOR_MULTIPLIER *
-                Constants.CILIA_ROTATION_FACTOR;
-        }
+        var effectiveMass = shape.GetMass();
```

Comment:
So this now ignores the shape of the microbe? Well that's probably fine as microbes are always centered, but taking actual angular velocity inertia into account would be a much more accurate calculation. I think right now this will calculate the same rotation speed for a thin and long stick as well as a small round blob, which I think should get vastly different speeds due to how far away the mass is from the center in the long thin stick.